include/units/core.h

```diff
@@ -2110,19 +2101,19 @@ namespace units
 		 *				`conversion_factor` (-Winjected-class-name).
 		 */
 		template<class UnitFrom, class UnitTo>
-		struct delayed_is_same_dimension : std::false_type
+		struct delayed_is_same_dimension_conversion_factor : std::false_type

```

Comment:
This is unused. If Clang 6 doesn't give warnings, as it documents (-Winjected-class-name), it can be removed.